onnxruntime_extensions/pnp/_torchext.py

```diff
@@ -268,4 +268,35 @@ def export(self, *args, **kwargs):
             return super().export(*args, **kwargs)
         else:
             oxml = core_m.export(*args, **kwargs)
-            return ONNXModelUtils.join_models(prefix_m, oxml)
+            model = ONNXModelUtils.join_models(prefix_m, oxml)
```

Comment:
ideally, the input_names should be in prefix_m only, and output_names is in oxml only. but the implementation is also since it is more straightforward.

Reply:
Apparently, the  join operation can change the node names to prevent name collisions. It's better to do this rename after the join operation.